python-sdk/tests/benchmark/Dockerfile

```diff
@@ -1,4 +1,4 @@
-FROM debian:bullseye AS build
+FROM --platform=linux/amd64 debian:bullseye AS build
```

Comment:
You can also pass this option to the `docker` command line which I think is what you want as not everyone uses the same system.

Reply:
See [docker build](https://docs.docker.com/engine/reference/commandline/build/)

Reply:
+1